src/components/layers-panel/layers-control-panel.tsx

```diff
@@ -69,6 +73,28 @@ export const LayersControlPanel = ({
   onPointToLayer,
   deleteLayer,
 }: LayersControlPanelProps) => {
+
+  // stub {
+  const dispatch = useAppDispatch();
+
+  const handleArcGisLogin = () => {
+    dispatch(arcGisLogin());
+  };
+  
+  const handleArcGisLogout = () => {
+    dispatch(arcGisLogout());
```

Comment:
Logout has "Are you sure?" step by Figma

Reply:
Done.